### PR TITLE
Clean up SocketTestServerAsync.cs.

### PR DESCRIPTION
**Pull request by pgavlin**
I missed this yesterday when doing cleanup work.

### CODE REVIEW

**Comment by pgavlin** [comment]
@CIPop @davidsh PTAL

File: src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs [hunk 1]
```diff
@@ -220,38 +220,36 @@ private void IO_Completed(object sender, SocketAsyncEventArgs e)
...
+                    break;
                 case SocketAsyncOperation.Send:
-                ProcessSend(e);
-                break;
+                    ProcessSend(e);
+                    break;
                 default:
-                throw new ArgumentException("The last operation completed on the socket was not a receive or send");
+                    throw new ArgumentException("The last operation completed on the socket was not a receive or send");
             }
```
Comment (davidsh) [inline src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs:230]: the spacing looks like extra tabs/spaces in various lines above.

**Comment by davidsh** [comment]
Minor comments, otherwise LGTM

File: src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs (hunk 1 above)
Comment (pgavlin) [inline src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs:230]: The spacing was off originally--I fixed it with this commit.

File: src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs [hunk 2]
```diff
@@ -76,24 +76,24 @@ protected override void Dispose(bool disposing)
         private void Init()
         {
             // Allocates one large byte buffer which all I/O operations use a piece of.  This gaurds
```
Comment (CIPop) [inline src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs:78]: Spelling: guards instead of gaurds

**Comment by pgavlin** [comment]
Agh, sorry--pulled the trigger too fast. Please keep reviewing, I'll address any comments with another PR.

File: src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs (hunk 2 above)
Comment (pgavlin) [inline src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs:78]: Done.

File: src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs [hunk 3]
```diff
@@ -220,38 +220,36 @@ private void IO_Completed(object sender, SocketAsyncEventArgs e)
             switch (e.LastOperation)
             {
                 case SocketAsyncOperation.Receive:
-                ProcessReceive(e);
-                break;
+                    ProcessReceive(e);
```
Comment (CIPop) [inline src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs:223]: Are you using the corefx VS settings? These were moved back when I've imported those in VS.
My personal preference is to have them indented as you have them here (as long as these are spaces and not tabs :) ).

**Comment by CIPop** [comment]
LGTM

File: src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs (hunk 3 above)
Comment (pgavlin) [inline src/Common/tests/System.Net/Sockets/SocketTestServerAsync.cs:223]: > Are you using the corefx VS settings?

This is what the formatter gave me, yes.

> as long as these are spaces and not tabs :)

Hah, yes--they are spaces this time around.

**Pull request merged**